rules/Renaming/NodeManipulator/ClassRenamer.php

```diff
@@ -139,7 +139,7 @@ private function refactorName(Name $name, array $oldToNewClasses): ?Name
 
     private function removeUseName(Name $oldName): void
     {
-        $uses = $this->betterNodeFinder->findFirstPreviousOfNode(
+        $uses = $this->betterNodeFinder->findFirstPrevious(
```

Comment:
What is the difference of this method? It seems they're doing the same thing. Maybe just one should exist

Reply:
To be honest, i don´t know why these both methods exists. There seem to be subtle differences.